app/src/main/java/com/github/marceltorne/meili/LoginActivity.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.marceltorne.meili
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.Button
+import android.widget.EditText
+import android.widget.TextView
+import android.widget.Toast
+import com.google.firebase.auth.FirebaseAuth
+
+class LoginActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login)
+
+
+
+        findViewById<Button>(R.id.login_button).setOnClickListener {
+            loginUser()
+        }
+
+        findViewById<TextView>(R.id.back_to_registration_text_view).setOnClickListener {
+            //Launch the login activity
+            finish()
+        }
+    }
+
+    private fun loginUser() {
```

Comment:
I think it would be a good idea to call this function a little bit different, for example "simpleLoginUser" because we will also have googleLogin and then it could be confusing. What do you think?